### PR TITLE
Refactor breadcrumbs

### PR DESCRIPTION
**Pull request by emilps**

### CODE REVIEW

File: webapp/src/components/Breadcrumbs/Breadcrumbs.jsx [hunk 1]
```diff
@@ -3,21 +3,22 @@ import React from 'react';
...
   return (
     <div className="breadcrumb container">
       <Link to="/" style={{ textDecoration: 'none' }}>
         <p className="breadcrumb main">Bærum</p>
       </Link>
-      {names[0] !== null && names[0] !== undefined ? (
-        <Link to={`/${numbers[0]}`}>
-          <p className="breadcrumb">/ {names[0]}</p>
+
+      {tractName !== null && tractName !== undefined ? (
```
Comment (mariusaarsnes) [inline webapp/src/components/Breadcrumbs/Breadcrumbs.jsx]: maybe split this into a check like `tractSelected` which simply returns the result of that check?

File: webapp/src/components/Breadcrumbs/Breadcrumbs.jsx [hunk 2]
```diff
@@ -3,21 +3,22 @@ import React from 'react';
...
+        <Link to={`/${tractNumber}`}>
+          <p className="breadcrumb">/ {tractName}</p>
         </Link>
       ) : null}
 
-      {names[1] !== null && names[1] !== undefined ? (
-        <Link to={`/${numbers[0]}/${numbers[1]}`}>
-          <p className="breadcrumb">/ {names[1]}</p>
+      {unitName !== null && unitName !== undefined ? (
```
Comment (mariusaarsnes) [inline webapp/src/components/Breadcrumbs/Breadcrumbs.jsx]: same as above

File: webapp/src/components/Breadcrumbs/Breadcrumbs.jsx (hunk 1 above)
Comment (emilps) [reply webapp/src/components/Breadcrumbs/Breadcrumbs.jsx]: This is now done by validateSelected

File: webapp/src/components/Breadcrumbs/Breadcrumbs.jsx (hunk 2 above)
Comment (emilps) [reply webapp/src/components/Breadcrumbs/Breadcrumbs.jsx]: Same as above

**Pull request merged**